### PR TITLE
Developer guide use case

### PR DESCRIPTION
**Pull request by prit3010**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -283,31 +283,73 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 **Use case: Delete a person**
 
 **MSS**
 
 1.  User requests to list persons
-2.  AddressBook shows a list of persons
+2.  Coydir shows a list of persons
 3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+4.  Coy deletes the person
```
Comment (kevinchangjk) [inline docs/DeveloperGuide.md:295]: What is `Coy`?

**Review by kevinchangjk — CHANGES_REQUESTED** [review]
As mentioned, probably need to change "Coy" to "Coydir". Otherwise LGTM.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (prit3010) [reply docs/DeveloperGuide.md:312]: In the Singapore Armed Forces, a Coy refers to a company

Comment (prit3010) [reply docs/DeveloperGuide.md:312]: Resolved

**Comment by prit3010** [comment]
Fixes #8

**Review by kevinchangjk — APPROVED** [review]
Okay very nice. Thanks for the explanation.

**Pull request merged**